build.sbt

```diff
@@ -6,7 +6,7 @@ organization := "com.azavea"
 
 organizationName := "Azavea"
 
-scalaVersion := "2.11.11"
+scalaVersion in ThisBuild := "2.11.11"
```

Comment:
What does this do?

Reply:
Prevents ensime from freaking out when I run `enimeConfig` every once in a while in sbt. You need to rerun it every time you add a new dependency.

Reply:
:+1: